cmd/prepare-storage-msg.go

```diff
@@ -51,12 +51,6 @@ func printOnceFn() printOnceFunc {
 	}
 }
 
-// Prints custom message when healing is required for XL and Distributed XL backend.
-func printHealMsg(endpoints []*url.URL, storageDisks []StorageAPI, fn printOnceFunc) {
```

Comment:
The following FIXME was kept here for a reason 

```
// Constructs a formatted heal message, when cluster is found to be in state where it requires healing.
// healing is optional, server continues to initialize object layer after printing this message.
// it is upto the end user to perform a heal if needed.
func getHealMsg(endpoints []*url.URL, storageDisks []StorageAPI) string {
        msg := fmt.Sprintln("\nData volume requires HEALING. Healing is not implemented yet stay tuned:")
        // FIXME:  Enable this after we bring in healing.
        //      msg := "mc admin heal myminio"
```

The startup sequence detects if disks need to be healed, we should show a message while using `mc admin heal` cli how it can be done.